packages/paste-website/src/pages/patterns/filter-group/index.mdx

```diff
@@ -145,20 +143,6 @@ Always consider opportunities to reduce cognitive load by limiting the number of
 
 <DefaultFilterGroupExample />
 
-#### Empty state
-
-When the applied filters and/or search does not return any results, use the [Empty State pattern](/patterns/empty-state) to inform the user and provide a method to reset all filter and search criteria.
-
-<EmptyStateFilterGroupExample />
-
-#### Custom date range
-
-While we don’t yet have an official custom date/time component, in the interim it is recommended to use the following combination of Popover, Date Input, and Time Input to allow customers to select a custom date/time range.
-
-When the Custom option is selected, the Popover containing the Date Inputs and Time Inputs necessary would be displayed to allow customers to enter their desired range. Once the Apply button is clicked, the Popover would hide. If a customer wants to change their original selections, they must click on the Calendar link style button to open the Popover again.
-
-<CustomDateFilterGroupExample />
-
```

Comment:
suggestion(blocking): we should preserve this documentation along with examples if we are not immediately updating Airtable and this still being served to consumers